tbconnect/models.py

```diff
@@ -87,3 +87,4 @@ class TBCheck(models.Model):
     timestamp = models.DateTimeField(default=timezone.now, db_index=True)
     risk = models.CharField(max_length=22, choices=RISK_CHOICES)
     follow_up_optin = models.BooleanField(default=False)
+    language = models.CharField(max_length=3, null=True, blank=True)
```

Comment:
Should we have a list of `choices` with the language codes that we expect?

Reply:
Would probably be useful, let me add it